server/app/models.py

```diff
@@ -1199,3 +1207,17 @@ def _can(cls, user, need, final, query):
     def _pre_put_hook(self):
         # TODO Remove when submitter is a computed property
         self.submitter = self.submission.get().submitter
+
+    def get_comp_score(self):
```

Comment:
I see two small issues with this design.

(1) I think it's easy to generalize this method to get all scores, instead of just composition. Remove the `score.tag == 'composition'` check and add a `tag` field to each response. So, this function should be called `get_scores` and it should return a bunch of tagged scores, each with a message and grader. 

(2) I'm confused why the `submitter` email is returned. When a student works with a partner, she may have a `FinalSubmission` for which she is not the submitter. That person still needs to be assigned a grade.